src/main/java/org/fundacionjala/pivotal/pages/Story.java

```diff
@@ -128,4 +159,56 @@ public boolean existStory(final String text) {
         return false;
     }
 
+    /**
+     * click dropdown story button..
+     *
+     * @param text name of the history
+     */
+    public void clickstoryDropdownButton(final String text) {
+        String xpathExpression = "//span[@data-aid=\"StoryPreviewItem__title\" "
+                .concat("and text()='").concat(text)
+                .concat("']/ancestor::header/child::button");
+        action.click(By.xpath(xpathExpression));
+    }
+
+    /**
+     * click delete button.
+     */
+    public void clickDeleteButton() {
+        this.action.click(this.deleteStoryButton);
+
+    }
+
+    /**
+     * click delete button.
+     */
+    public void clickConfirmDeleteButton() {
+        this.action.click(this.deleteStoryConfirmButton);
+
+    }
+
+    /**
+     * select an especific checkbox.
+     *
+     * @param text is the name of a speceif story .
+     */
+    public void clickStoryCheckboxButton(final String text) {
+        for (final WebElement element : this.storyCheckBoxes) {
+            element.click();
+            clickDeleteButtonOfToast();
```

Comment:
don't forget refactor it creating DeleteDialog page, check if this dialog is used in workspace and project, here or in new branch

Reply:
fixed with commit f8c385f1d168181e31b575f1cf3c3229e7c6c2b0